rocAL/rocAL_pybind/examples/rocAL_api_caffe_reader.py

```diff
@@ -113,19 +114,18 @@ def main():
                     draw_patches(image_batch[element], cnt)
             data_loader.reset()
         else:
-            for i, (image_batch, bboxes, labels) in enumerate(data_loader, 0):  # Detection
-                if i ==0 :
+            for i, ([image_batch], bboxes, labels) in enumerate(data_loader, 0):  # Detection
+                if i == 0:
                     if args.print_tensor:
                         sys.stdout.write("\r Mini-batch " + str(i))
                         print("Images", image_batch)
                         print("Bboxes", bboxes)
                         print("Labels", labels)
                 for element in list(range(batch_size)):
                     cnt = cnt + 1
-                    draw_patches(image_batch[element], cnt)
+                    draw_patches(image_batch[element], cnt, bboxes[element])
             data_loader.reset()
-    print('Finished Training !!')
     print("##############################  CAFFE READER (CLASSIFCATION/ DETECTION)  SUCCESS  ############################")
 
-if __name__ == '__main__':
+if __name__ == "__main__":
```

Comment:
Why is this change done

Reply:
I used a python formatter called autopep8 which changes all ' to ". I'll revert these changes at once